src/main/webapp/app/exercises/programming/manage/programming-exercise-detail.component.html

```diff
@@ -200,6 +189,22 @@ <h2><span jhiTranslate="artemisApp.programmingExercise.detail.title">Programming
                         <span jhiTranslate="artemisApp.programmingExercise.lockAllRepositories">Lock all repositories</span>
                     </button>
 
+                    <button
+                        *ngIf="programmingExercise.isAtLeastInstructor"
```

Comment:
It's not visible to the editor anymore. Is this on purpose? If so, I don't think this is reflected in the documentation.
Also why?

Reply:
Good catch, that was a copy-paste error and not intended.